src/lib/connectors/repos/scheduled-notifications.js

```diff
@@ -26,8 +27,36 @@ const update = (id, changes) => helpers.update(ScheduledNotification, 'id', id,
 const findByEventId = eventId =>
   helpers.findMany(ScheduledNotification, { event_id: eventId });
 
+/**
+ * Finds scheduled notifications by licence number
+ * Checks if messages are delivered/received in Notify
+ *
+ * @param {String} licenceNumber
+ * @param {Number} page
+ * @param {Number} perPage
+ * @returns
+ */
+const findByLicenceNumber = async (licenceNumber, page, perPage) => {
```

Comment:
Something is a bit weird here but I'm not sure what it is. Have a look at the screenshot below from the UI
![image](https://user-images.githubusercontent.com/6029895/118610238-75e67a80-b7b3-11eb-9079-1a51f5e9685d.png)

This value is coming out blank for me. Am I doing something wrong? Or have we accidentally changed one of the variable names which no longer match between the service and the UI?

Reply:
Edit: It could just be correct.

Reply:
There was a bug where pre-2008 returns were displayed - fixed now